fanuc_m20id_support/package.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>fanuc_m20id_support</name>
+  <version>0.5.1</version>
+  <description>
+    <p>
+      ROS-Industrial support for the Fanuc M-20iD (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Fanuc M-20iD manipulators. This currently includes the /25 model only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>M-20iB/25 - "Cable integrated J3 Arm"</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information in
+      the <em>FANUC Robot M-20iB Mechanical Unit Operator's Manual</em> version
+      <em>FEC-RO-DS-M20iD25-DE-1</em>. All urdfs are based on the default motion and
```

Comment:
More copy-pasta?

Let's use the international documentation as much as possible:

```suggestion
      the <em>FANUC Robot M-20iD Mechanical Unit Operator's Manual</em> version
      <em>B-84074EN/08</em>. All urdfs are based on the default motion and
```

I've also used that manual to verify limits etc.